config/locales/en.yml

```diff
@@ -297,6 +297,7 @@ en:
       user_exists: "Sorry, %{username} is not an existing user."
       invalid_language_or_project: Invalid language/project
     expected_students: Expected Students
+    expected_students_doc: Number of expected students count
```

Comment:
This is redundant. Remove the count at the end or change to something like 'Expected students count'